rnaseq-star-align/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM python:3.6-alpine
+
+ARG WORKFLOW
+RUN apk add make
+
+COPY ./${WORKFLOW} /opt 
+
+RUN make -C /opt init-pip
+
+COPY tools/ /tools
+COPY scripts/ /scripts
+
+COPY rnaseq/ /opt
```

Comment:
Could consider better ways to build GPAS workflows on top of workflows for external users.